libc/test/include/stdbit_test.cpp

```diff
@@ -0,0 +1,22 @@
+//===-- Unittests for stdbit ----------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDSList-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include "test/UnitTest/Test.h"
+
+#include "llvm-libc-macros/stdbit-macros.h"
+#include "src/__support/CPP/limits.h" // UINT_WIDTH
+#include "src/stdbit/stdc_leading_zeros_uc.h"
+#include "src/stdbit/stdc_leading_zeros_ui.h"
+#include "src/stdbit/stdc_leading_zeros_ul.h"
+#include "src/stdbit/stdc_leading_zeros_ull.h"
+#include "src/stdbit/stdc_leading_zeros_us.h"
+
+TEST(LlvmLibcStdbitTest, TypeGenericMacro) {
+  using namespace LIBC_NAMESPACE;
```

Comment:
generally in tests we should avoid doing `using namespace LIBC_NAMESPACE` since it means that if the namespaced symbol isn't available but the system one is, the test will still pass.

Reply:
I've changed this test significantly in 338637753432, PTAL